_source/_posts/2022-07-20-express-prisma-tailwind.md

```diff
@@ -0,0 +1,665 @@
+
+### Create the CSS file using Tailwind CSS
+
+The last part of this process is to create the actual CSS file that your application will use.  In your `public/stylesheets` directory, you'll see a file called `style.css`.  Replace the code in that file with the code below.
+
+```css
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+body {
+  padding: 50px;
+  font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
+}
+
+a {
+  color: #00B7FF;
+}
+```
+
+This is largely the same. However, you are importing the directives for each of Tailwind's layers.
+
+Next, you can compile the classes you used in your pug files and the style sheet you just modified to create a new CSS file. Run the following command.
+
+```bash
+npx tailwindcss -i ./public/stylesheets/style.css -o ./public/stylesheets/output.css --watch
+```
+
+Now you can open the `output.css` file and see the complete Tailwind CSS code.  The `--watch` parameter here is helpful for local development.  This will rebuild the `output.css` file each time you change one of your layouts.  You can disregard that parameter if you do not want to make any changes.
+
+## Test your application
+
+You're now ready to start your application.  Run the following command.
+
+```bash
+npm run start
+```
+
+Once your application is started you should be able to navigate to *http://localhost:3000/* and see your home page.
+
+{% img blog/express-prisma-and-tailwindcss/HomePage.PNG alt:"Home page" width:"800" %}{: .center-image }
+
+Use the login button at the top of the screen and navigate out to the Okta login page.  Once you've logged in you will be redirected back to your website where you will be able to see your dashboard page.
+
+{% img blog/express-prisma-and-tailwindcss/DashboardPage.PNG alt:"Dashboard page" width:"800" %}{: .center-image }
+
+## What you learned in this article
+
+In this article, you learned how to use Prisma to define a schema for your database.  You then learned how to create a migration from that schema and seed your database.  You saw that Prisma could be used with the most popular database options, including SQL Server, MySql, MongoDB, and more.  
+
+You also learned how to build a CSS file with Tailwind CSS.  You learned the simple tailwind configuration and how the Tailwind CLI uses your code to build its CSS file.
+
+Finally, you learned how to secure your express application using passport.
+
+## Learn More About Node, Authentication, and Okta
+
+Can't get enough of Node? Check out our [quickstarts for Node](https://developer.okta.com/code/nodejs/) and other excellent posts from the Okta Developer blog:
+- [Build and Deploy a Node.js App to Heroku](/blog/2022/02/28/build-deploy-node-app-heroku)
+- [Node.js Login with Express and OIDC](/blog/2020/06/16/nodejs-login)
+- [Build a Node.js API with TypeScript](/blog/2019/05/07/nodejs-typescript-api)
```

Comment:
We seem to be missing the usual closing boilerplate... :

"Make sure you follow us on [Twitter](https://twitter.com/oktadev) and subscribe to our [YouTube](https://www.youtube.com/c/oktadev) channel. Please comment below if you have any questions or want to share what tutorial you’d like to see next."